src/frequenz/sdk/timeseries/battery_pool/battery_pool.py

```diff
@@ -373,6 +378,24 @@ def soc(self) -> MetricAggregator[Sample[Percentage]]:
 
         return self._active_methods[method_name]
 
+    @property
+    def temperature_max(self) -> MetricAggregator[Sample[Temperature]]:
```

Comment:
I think we need some sort of convention for metrics that have a max/min/avg, etc. Adding `temperature_min`, `temperature_avg`, etc. doesn't look super nice. A few alternatives that come to mind:

* `pool.temperature.max`: A bit annoying as we need to add an aggregation (some extra class)
* `pool.temperature(MAX)`: A bit annoying that it is not a property like the other metrics

This might also apply to apparent/real/reactive/active power for example, which we have discussed in the last SDK meeting.

Reply:
I've just pushed over it, i went out of draft one push to early

Reply:
OK, I gave this another look but GitHub said the diff was updated again while I was writing comments, so I will back off now, please let us know when the PR has settled and it is ready for a review :)